src/bootstrap/doc.rs

```diff
@@ -582,13 +582,20 @@ impl Step for Rustc {
 
         // Build cargo command.
         let mut cargo = builder.cargo(compiler, Mode::Rustc, SourceType::InTree, target, "doc");
+
+        cargo.rustdocflag("-Zunstable-options");
         cargo.rustdocflag("--document-private-items");
         // Since we always pass --document-private-items, there's no need to warn about linking to private items.
         cargo.rustdocflag("-Arustdoc::private-intra-doc-links");
         cargo.rustdocflag("--enable-index-page");
         cargo.rustdocflag("-Zunstable-options");
         cargo.rustdocflag("-Znormalize-docs");
         cargo.rustdocflag("--show-type-layout");
+
+        // Explicitly link to extern docs for relevant crates
+        cargo.rustdocflag("--extern-html-root-url");
+        cargo.rustdocflag("ena=https://docs.rs/ena/0.14.0/");
```

Comment:
is it ok to be using the version number in here?

Reply:
Oh good point - it is ok to use the version number but means we have to update it manually when the dependency gets bumped. -Zrustdoc-map will take care of that automatically.